proxy/coco/src/error.rs

```diff
@@ -56,6 +56,10 @@ pub enum Error {
     #[error(transparent)]
     Io(#[from] io::Error),
 
+    /// Kv error.
+    #[error(transparent)]
+    Kv(#[from] kv::Error),
```

Comment:
Where did we land on sub-error enums? :eyes:

Reply:
We want them and we gonna address them in #850.

Reply:
Muchos gracias